packages/card-support/src/components/DownloadClientData/utils.tsx

```diff
@@ -141,7 +141,8 @@ export const submitForm = async (
             first_name: client.firstName,
             last_name: client.lastName,
             gender: client.gender,
-            facility_of_registration: location ? location.title : '',
+            // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+            facility_of_registration: client.attributes.registration_location_name || '',
```

Comment:
would making the attribute optional in the type [here](https://github.com/opensrp/web/blob/42c222c444b0f40ba03953e880daa69a760e1693/packages/card-support/src/ducks/clients.ts#L38) avoid disabling the rule here.